models/service.js

```diff
@@ -27,6 +27,29 @@ module.exports = function(sequelize, DataTypes) {
       type: DataTypes.BOOLEAN,
       allowNull: false,
       defaultValue: true
+    },
+    rating: {
+      type: DataTypes.FLOAT,
+      allowNull: false,
+      defaultValue: 0.0
```

Comment:
a value of rating = 0 means rating is 0 while `null` means value is not available. I think we should allow null here.